openbb_platform/providers/oecd/openbb_oecd/models/unemployment.py

```diff
@@ -0,0 +1,123 @@
+"""OECD Unemployment Model."""
+
+
+import re
+from datetime import date, timedelta
+from typing import Any, Dict, List, Literal, Optional, Union
+
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.unemployment import (
+    UnemploymentData,
+    UnemploymentQueryParams,
+)
+from openbb_oecd.utils import constants, helpers
+from pydantic import Field, field_validator
+
+countries = tuple(constants.COUNTRY_TO_CODE_UNEMPLOYMENT.keys())
+CountriesLiteral = Literal[countries]  # type: ignore
+
+
+class OECDUnemploymentQueryParams(UnemploymentQueryParams):
+    """OECD Unemployment Query."""
+
+    country: CountriesLiteral = Field(
+        description="Country to get unemployment for.", default="united_states"
+    )
```

Comment:
Additional parameters are:

Sex: [Female, Male, Total]

Age: [Total, 15 to 19, 15 to 24, 15 to 64, 20 to 24, 25 to 29, 25 to 34, 25 to 54, 25 to 64, 30 to 34, 35 to 39, 34 to 44, 40 to 44, 45 to 49, 45 to 54, 50 to 54, 55 to 59, 55 to 64, 60 to 64, 65 to 69, 65+, 70+]

Reply:
Intentionally left out for scope here.